flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliClient.java

```diff
@@ -613,6 +624,20 @@ private void printInfo(String message) {
 		terminal.flush();
 	}
 
+	/**
+	 * Output Results in non-interactive way.
+	 *
+	 * @return
+	 */
+	void printRows(final ResultDescriptor desc, long start, long end) {
```

Comment:
Give method a more meaningful name such as `printNonInteractiveResult`. Then the Javadoc is not required anymore.

Reply:
OK